proposals/FileSystemSubscriptionManager.md

```diff
@@ -0,0 +1,123 @@
+# Proposal: FileSystemSubscriptionManager
+
+## Authors
+
+* Ming-Ying Chung (<mych@google.com>)
+* Austin Sullivan (<asully@google.com>)
```

Comment:
nit: please use my chromium email account (asully@chromium.org) and list my affiliation as Google. Thanks!

Reply:
Updated to link to github handle instead.